.eslintignore

```diff
@@ -1,2 +0,0 @@
-node_modules
```

Comment:
can probably also come back as well, although I am not 100% sure if it makes any difference 🤷